client/locations-list-map.js

```diff
@@ -1090,6 +1096,9 @@ function getEntryEl(entry) {
     sendEvent('listView', 'clickZoom', entry.name);
     zoomToMarker(entry.marker);
   });
+  $(entry.domElem).find('.entry-partner-link').on('click', () => {
+    window.open(`${document.body.dataset.partnerSite}ftm?id=${entry.row}`, '_blank');
```

Comment:
Is it worth having some config allow for partners to declare their own callback URL?  Or is that oo much for now?

Reply:
we could just make the `linkPartners` from `applicationRoutes.js` (abstracted to another file) into an object keyed by the referrer, values being the callback URL?

Reply:
As @ngottlieb says, it wouldn't be hard to do. However, the original request as it filtered to me sounded maybe like partners were saying "Can you tell us how to ... ?".  You all know the partners and I don't-- your call on whether simple/standard or minor complexity/flexible is better. Or both: allow custom callback but default to `/ftm?id=X` when not specified.